src/main/java/com/synopsys/integration/detect/lifecycle/run/step/ContainerScanStepRunner.java

```diff
@@ -79,6 +78,7 @@ public Optional<UUID> invokeContainerScanningWorkflow() {
             uploadImageMetadataToStorageService();
             operationRunner.publishContainerSuccess();
             logger.info("Container scan image uploaded successfully.");
+            codeLocationName = createContainerScanCodeLocationName();
```

Comment:
`initiateScan()` which is called a bit earlier uses this variable. Is this not going to be a problem?

Reply:
Thanks @andrian-synopsys - yes I agree this should ideally be called before scan initiation. This wasn't causing any problems in any of the scan steps (but we were sending an empty code location name in the protobuf header). Addressed this in #973, thanks.